back/engines/commercial/report_builder/app/models/report_builder/report.rb

```diff
@@ -49,6 +51,8 @@ class Report < ::ApplicationRecord
     validates :name, uniqueness: true, allow_nil: true
     validates :phase_id, uniqueness: true, unless: :supports_multiple_phase_reports?, allow_nil: true
     validates :visible, inclusion: { in: [false], unless: :phase? }
+    validates :year, numericality: { in: 2024..2050 }, allow_nil: true
+    validates :quarter, numericality: { in: 1..4 }, allow_nil: true
```

Comment:
2050 - should we extend this to something even further in the future maybe? Or do you think this is fine? :)

Reply:
I was thinking that if this code is still around in 25 years we might be in trouble!

Reply:
Haha very true :joy: :+1: